src/components/dataset/table.vue

```diff
@@ -50,4 +53,16 @@ const { datasets } = useRequestData();
     padding-right: 10%;
   }
 }
+.empty-table-message {
+  margin-bottom: 20px;
+  margin-top: -20px;
```

Comment:
Does this mean that there won't be space between the column headers of the table and the `.empty-table-message`? We usually leave some space between the two.

Reply:
I was following Forms list and it doesn't have space. Now I have added margin for Datasets list as well as Forms list

![image](https://github.com/getodk/central-frontend/assets/447837/19195c48-1a0c-4c9b-9872-51525534a89f)

Reply:
Ah great, I'd been thinking that it'd be great to get space above the forms list message.